src/styles/cards.scss

```diff
@@ -0,0 +1,122 @@
+.cards {
+  padding: 50px 40px 72px;
+  display: grid;
+  gap: 46px 48px;
+  justify-content: center;
+
+  @include set-columns(1);
+
+  @media (min-width: 488px) {
+    @include set-columns(2);
+  }
+
+  @media (min-width: 768px) {
+    @include set-columns(3);
+  }
+
+  @media (min-width: 1024px) {
+    @include set-columns(4);
+  }
+
+  &__card {
+    width: 200px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 32px 16px 16px;
+
+    @include smooth-transition;
+
+    &:hover {
+      transform: scale(1.2);
+    }
```

Comment:
The hover effect causes the element to scale up, which can cause layout shifts. Ensure that this transformation does not affect the layout of surrounding elements, possibly by using a transform-origin or containing the size increase within the grid item.